x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/export_prebuilt_rule.cy.ts

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { bulkExportRules } from '../../../../tasks/rules_bulk_actions';
+import { exportRuleFromDetailsPage } from '../../../../tasks/rule_details';
+import {
+  expectedExportedRule,
+  expectedExportedRules,
+  getIndexPatterns,
+  getNewRule,
+} from '../../../../objects/rule';
+import {
+  exportRule,
+  filterByCustomRules,
+  filterByElasticRules,
+  selectAllRules,
+  selectRulesByName,
+} from '../../../../tasks/alerts_detection_rules';
+import { RULE_NAME, TOASTER_BODY } from '../../../../screens/alerts_detection_rules';
+import { createRuleAssetSavedObject } from '../../../../helpers/rules';
+import { deleteAlertsAndRules } from '../../../../tasks/api_calls/common';
+import { createAndInstallMockedPrebuiltRules } from '../../../../tasks/api_calls/prebuilt_rules';
+import { createRule, findRuleByRuleId, patchRule } from '../../../../tasks/api_calls/rules';
+
+import { login } from '../../../../tasks/login';
+
+import { visitRulesManagementTable } from '../../../../tasks/rules_management';
+
+const PREBUILT_RULE_ID = 'rule_1';
+
+describe(
+  'Detection rules, Prebuilt Rules Export workflow - With Rule Customization',
+  {
+    tags: ['@ess', '@serverless', '@skipInServerlessMKI'],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'prebuiltRulesCustomizationEnabled',
+          ])}`,
+        ],
+      },
+    },
+  },
+  () => {
+    describe('Rule export workflow with single rules', () => {
+      const PREBUILT_RULE = createRuleAssetSavedObject({
+        name: 'Non-customized prebuilt rule',
+        rule_id: PREBUILT_RULE_ID,
+        version: 1,
+        index: getIndexPatterns(),
+      });
+
+      beforeEach(() => {
+        login();
+        deleteAlertsAndRules();
+        cy.intercept('POST', '/api/detection_engine/rules/_bulk_action').as('bulk_action');
+        /* Create a new rule and install it */
+        createAndInstallMockedPrebuiltRules([PREBUILT_RULE]);
+        createRule(
+          getNewRule({ name: 'Custom rule to export', rule_id: 'custom_rule_id', enabled: false })
+        ).as('customRuleResponse');
+        visitRulesManagementTable();
+      });
```

Comment:
Same idea: we could set up the test data (rule assets, rules) only once for the whole test file, and then in each test only do the "act" (export) and "assert" phases.